Controller/Rest/ContentController.php

```diff
@@ -0,0 +1,288 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformUIBundle\Controller\Rest;
+
+use eZ\Publish\API\Repository\Exceptions\NotFoundException;
+use eZ\Publish\API\Repository\Values\Content\ContentCreateStruct;
+use eZ\Publish\API\Repository\Values\User\UserUpdateStruct;
+use eZ\Publish\Core\Repository\Values\User\UserCreateStruct;
+use eZ\Publish\Core\REST\Server\Exceptions\BadRequestException;
+use eZ\Publish\Core\REST\Server\Values\RestContent;
+use eZ\Publish\Core\REST\Server\Values\Version;
+use Symfony\Component\HttpFoundation\Request;
+use eZ\Publish\Core\REST\Common\Message;
+use eZ\Publish\Core\REST\Server\Controller\Content;
+use eZ\Publish\Core\REST\Server\Values\CreatedContent;
+use eZ\Publish\Core\REST\Server\Values\NoContent;
+use eZ\Publish\API\Repository\Values\User\User;
+use eZ\Publish\Core\FieldType\User\Value as UserFieldValue;
+
+/**
+ * Override of the REST Content Controller.
+ *
+ * Features:
+ * - Proxies Content calls on User content to the UserService.
+ */
+class ContentController extends Content
+{
+    private $settings = ['user-contenttype-id' => 4];
+
+    public function setSettings(array $settings)
```

Comment:
Guess this is inherited from parent, if not this could be a array of user content type ids.